helios-services/src/test/java/com/spotify/helios/rollingupdate/RollingUpdateOpFactoryTest.java

```diff
@@ -380,4 +389,43 @@ public void testYield() {
     assertEquals(0, op.operations().size());
     assertEquals(0, op.events().size());
   }
+
+  @Test
+  public void testErrorWhenIgnoreFailuresIsTrue() {
+    final DeploymentGroup deploymentGroup = DeploymentGroup.newBuilder()
+        .setName("ignore_failure_group")
+        .setRolloutOptions(RolloutOptions.newBuilder()
+            .setIgnoreFailures(true)
+            .build()
+        )
+        .setRollingUpdateReason(MANUAL)
+        .build();
+
+    // the current task is the AWAIT_RUNNING one
+    final DeploymentGroupTasks tasks = DeploymentGroupTasks.newBuilder()
+        .setTaskIndex(2)
+        .setRolloutTasks(ImmutableList.of(
+            RolloutTask.of(RolloutTask.Action.UNDEPLOY_OLD_JOBS, "host1"),
+            RolloutTask.of(RolloutTask.Action.DEPLOY_NEW_JOB, "host1"),
+            RolloutTask.of(RolloutTask.Action.AWAIT_RUNNING, "host1")
+        ))
+        .setDeploymentGroup(deploymentGroup)
+        .build();
+
+    final RollingUpdateOpFactory opFactory = new RollingUpdateOpFactory(tasks, eventFactory);
+
+    final RollingUpdateOp nextOp = opFactory.error("something went wrong", "host1",
+        RollingUpdateError.TIMED_OUT_WAITING_FOR_JOB_TO_REACH_RUNNING);
+
+    assertThat(nextOp.operations(), containsInAnyOrder(
+        new SetData("/status/deployment-groups/ignore_failure_group",
```

Comment:
You can use `Paths.statusDeploymentGroup("ignore_failure_group")` here and `Paths.statusDeploymentGroupTasks(` for line 428.